tests/coverage/read.me

```diff
@@ -0,0 +1,15 @@
+# gcovr -- generate GCC code coverage reports
+# Gcovr provides a utility for managing the use of the GNU gcov utility and generating summarized code coverage results. 
+# This command is inspired by the Python coverage.py package, which provides a similar utility for Python.
+
+# source code: https://github.com/gcovr/gcovr/
+# docs: https://gcovr.com/en/stable/guide.html
+# install GCOVR
+# $ pip install gcovr
+
+How to use.
+1) Run script:
+   ~/facelift/build$ tests/coverage/test_gcovr.sh
+2) After build, a test-case will be launched. 
```

Comment:
what should I do at that step 2 ??
I only see one step, which is to execute the script.

Reply:
It is just info for user. Not steps, but stages.
I agree with you, it is necessary to reduce/rewrite.